gui/irodsInfo.py

```diff
@@ -46,9 +46,10 @@ def refresh_info(self):
         self.versionLabel.setText(
             '.'.join((str(num) for num in self.ic.session.server_version)))
         # irods resources
-        names, spaces = self.ic.list_resources()
-        self.rescTable.setRowCount(len(names))
-        for row, (name, space) in enumerate(zip(names, spaces)):
+        parent_resources = [(resc, self.ic.resources[resc]['status'], self.ic.resources[resc]['free_space']) 
+                            for resc in self.ic.resources if self.ic.resources[resc]['parent'] == None]
+        self.rescTable.setRowCount(len(parent_resources))
+        for row, (name, status, space) in enumerate(parent_resources):
```

Comment:
It is better simply to update `list_resources()` here.  Simply add `status` to the tuple in both locations and avoid the cumbersome `parent_resources` assignment.

Reply:
It might be interesting for `list_resources()` to take a list of attributes to return in the tuple, defaulting to the name only.  Then here, you can simply call it with the list `['name', 'status', 'free_space']`.